services/eventService/eventService.go

```diff
@@ -1,43 +1,250 @@
 package eventService
 
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/patrickmn/go-cache"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"time"
+)
+
 // An Event represents a HackSoc event.
 type Event struct {
-	Name        string
-	Description string
-	Location    string
-	Date        string
-	URL         string
-	ImageURL    string
-	Upcoming    bool
+	Name            string
+	Description     string
+	Location        string
+	URL             string
+	ImageURL        string
+	AttendingCount  int
+	InterestedCount int
+	StartTime       time.Time
+	EndTime         time.Time
+}
+
+func (e *Event) GetInterestLine() string {
```

Comment:
How about `GetParticipation` or `GetInterestCount`?

Reply:
- GetParticipation doesn't include interest
- GetInterestCount is not correct, we are returning a string not a line.
Opinion:
- GetInterestLine should stay